analyzers/its/expected/akka.net/S3247-Akka.Tests.Shared.Internals-netstandard2.0.json

```diff
@@ -0,0 +1,10 @@
+{
+  "Issues": [
+    {
+      "Id": "S3247",
+      "Message": "Replace this type-check-and-cast sequence with an \u0027as\u0027 and a null check.",
+      "Uri": "https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/its/Projects/akka.net/src/core/Akka.Tests.Shared.Internals/Helpers/XAssert.cs#L56",
```

Comment:
Why was this not raised before?
`T` in this context is nullable so it cannot be a value type.

Is the semantic model not able to detect that `T` is a class?

Reply:
Same as above - NamedTypeSymbol check was removed on the way